tools/container.bzl

```diff
@@ -1,11 +1,48 @@
-load("@io_bazel_rules_docker//container:container.bzl", "container_push")
+load("@aspect_bazel_lib//lib:transitions.bzl", "platform_transition_filegroup")
+load("@rules_oci//oci:defs.bzl", "oci_push", "oci_image", "oci_image_index")
+load("@rules_pkg//pkg:tar.bzl", "pkg_tar")
+
+def multiarch_go_image(name, binary):
+    """Create a container image with two variants of the given go_binary target.
+    
+    Args:
+        name: resulting oci_image_index target
+        binary: label of a go_binary target; it may be transitioned to another architecture
+    """
+    images = []
+    tar_target = "_{}.tar".format(name)
+    image_target = "_{}.image".format(name)
+    pkg_tar(
+        name = tar_target,
+        srcs = [binary],
+        package_dir = "/app/cmd/bb_storage",
```

Comment:
Considering that not all our executables are named "bb_storage", I think we should remove this.

Can't we simply use /app as the package dir?

Reply:
Sure, as long as the only reference to the path within the container is the entry_point. If users might have some config that has this path, it's obviously a breaking change to put the binary somewhere different.

Reply:
It should be. I don't think it's typical for people to override the entrypoint for the Buildbarn containers.

Reply:
okay, changed to just `app`